trustymail/trustymail.py

```diff
@@ -260,20 +253,20 @@ def dmarc_scan(domain):
                 elif tag == "p":
                     domain.dmarc_policy = tag_dict[tag]
 
-    except DNSError as error:
+    except (dns.resolver.NoAnswer, dns.exception.Timeout, dns.resolver.NXDOMAIN) as error:
         handle_error("[DMARC]", domain, error)
 
 
 def find_host_from_ip(ip_addr):
-    return DNS.revlookup(ip_addr)
+    return str(resolver.query(reversename.from_address(ip_addr), "PTR")[0])
```

Comment:
Personally, I'm a fan of splitting major indices out, to clarify what they are. Maybe something like:

```python
hostname, _ =  resolver.query(reversename.from_address(ip_addr), "PTR")
return str(hostname)
```

?

Reply:
Also, does `reversename.from_address(ip_addr)` not provide a hostname itself?

Reply:
First point:  I agree and will make the change.

Second point:  No, it does not:
```python
Python 3.6.3 (default, Oct 24 2017, 14:48:20) 
[GCC 7.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import dns.resolver
>>> import dns.reversename
>>> record = dns.resolver.query("www.npr.org", "A")
>>> print(record.response)
id 39913
opcode QUERY
rcode NOERROR
flags QR RD RA
;QUESTION
www.npr.org. IN A
;ANSWER
www.npr.org. 172 IN CNAME wildcard.npr.org.edgekey.net.
wildcard.npr.org.edgekey.net. 20988 IN CNAME e4539.g.akamaiedge.net.
e4539.g.akamaiedge.net. 19 IN A 23.46.53.110
;AUTHORITY
;ADDITIONAL
>>> n = dns.reversename.from_address("23.46.53.110")
>>> print(n)
110.53.46.23.in-addr.arpa.
>>> 
```